css/owned-list-pages.styl

```diff
@@ -8,17 +8,34 @@
   .resources-container
     background: #ebebeb
     padding: 0 3vw
+    text-align: center
```

Comment:
Now that owned card is collections/favourites etc only and not used in project cards, we should not be changing this file... ( I think?)